### PR TITLE
Add search functionality to endpoint dropdown

### PR DESCRIPTION
**Pull request by simon-johansson**
First commit 🎉🍾

## What
* Convert the endpoint dropdown into a Combobox with fuzzy search possibility
* Bonus: Add prettier(https://prettier.io/) for automatic code formatting. This also allows for sorting the tailwind css classes so that they automagically gets displayed in the same order everywhere. 

## Why
This was something that Skrym requested as it can be cumbersome to find the endpoint you are looking for if you have a lot of them.

## How
* Use Combobox component from HeadlessUI (https://headlessui.com/react/combobox#combobox)
* Add fuzzysort (https://www.npmjs.com/package/fuzzysort) npm dependency

## What is left
* Have not formatted all files according to the prettier rules.
* Would like to add basic unit tests to the Combobox component, but we need to add a unit testing library in order to do that.

## Screenshot
fuzzy search

### CODE REVIEW

**Review by eandre — DISMISSED** [review]

File: cli/daemon/dash/dashapp/src/components/Combobox.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  }, [wrapperRef]);
+
+  const filteredItems = fuzzysort.go(query, items, {
+    key: "name",
+    all: true,
+    threshold: -200,
+  });
+
+  return (
+    <HeadlessCombobox
```
Comment (DomBlack) [inline cli/daemon/dash/dashapp/src/components/Combobox.tsx:43]: Does the HeadlessCombobox handle aria labels etc? We use them on the webapp, would be nice to start using them everywhere to improve the accessibility

**Review by DomBlack — COMMENTED** [review]
🎉 Awesome first PR!

File: cli/daemon/dash/dashapp/src/components/Combobox.tsx (hunk 1 above)
Comment (simon-johansson) [reply cli/daemon/dash/dashapp/src/components/Combobox.tsx:43]: Thanks! 
Yes, they add some aria stuff:
image

They also add the functionality to select via keyboard and tab so it's not mouse/trackpad dependent. 
https://headlessui.com/react/combobox#accessibility-notes

Comment (DomBlack) [reply cli/daemon/dash/dashapp/src/components/Combobox.tsx:43]: Awesome! 🙏

**Review by DomBlack — APPROVED** [review]

**Review by eandre — APPROVED** [review]

**Pull request merged**